<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="enabled">
            <bool>true</bool>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>436</width>
                <height>421</height>
            </rect>
        </property>
        <property name="font">
            <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
            </font>
        </property>
        <property name="acceptDrops">
            <bool>false</bool>
        </property>
        <property name="windowTitle">
            <string>Калькулятор</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>calculator.svg</normaloff>calculator.svg
            </iconset>
        </property>
        <property name="statusTip">
            <string/>
        </property>
        <property name="autoFillBackground">
            <bool>true</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>9</x>
                    <y>10</y>
                    <width>411</width>
                    <height>401</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QTextBrowser" name="textBrowser">
                        <property name="enabled">
                            <bool>true</bool>
                        </property>
                        <property name="font">
                            <font>
                                <pointsize>14</pointsize>
                                <weight>75</weight>
                                <italic>true</italic>
                                <bold>true</bold>
                                <strikeout>false</strikeout>
                                <kerning>true</kerning>
                            </font>
                        </property>
                        <property name="cursor" stdset="0">
                            <cursorShape>ArrowCursor</cursorShape>
                        </property>
                        <property name="mouseTracking">
                            <bool>false</bool>
                        </property>
                        <property name="tabletTracking">
                            <bool>false</bool>
                        </property>
                        <property name="autoFillBackground">
                            <bool>false</bool>
                        </property>
                        <property name="readOnly">
                            <bool>true</bool>
                        </property>
                        <property name="html">
                            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style
                                type=&quot;text/css&quot;&gt;
                                p, li { white-space: pre-wrap; }
                                &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2';
                                font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;
                                &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px;
                                margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt;
                                font-weight:400; font-style:normal;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                            </string>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QGridLayout" name="gridLayout_3">
                        <item row="4" column="0">
                            <widget class="QPushButton" name="pushButton_delete">
                                <property name="text">
                                    <string>CC</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QPushButton" name="pushButton_4">
                                <property name="text">
                                    <string>4</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="3">
                            <widget class="QPushButton" name="pushButton_addition">
                                <property name="text">
                                    <string>+</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="3">
                            <widget class="QPushButton" name="pushButton_equally">
                                <property name="text">
                                    <string>=</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QPushButton" name="pushButton_7">
                                <property name="text">
                                    <string>7</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="2">
                            <widget class="QPushButton" name="pushButton_comma">
                                <property name="text">
                                    <string>,</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <widget class="QPushButton" name="pushButton_division">
                                <property name="text">
                                    <string>/</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="3">
                            <widget class="QPushButton" name="pushButton_minus">
                                <property name="text">
                                    <string>-</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QPushButton" name="pushButton_2">
                                <property name="text">
                                    <string>2</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="2">
                            <widget class="QPushButton" name="pushButton_3">
                                <property name="text">
                                    <string>3</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QPushButton" name="pushButton_8">
                                <property name="text">
                                    <string>8</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="2">
                            <widget class="QPushButton" name="pushButton_6">
                                <property name="text">
                                    <string>6</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="0">
                            <widget class="QPushButton" name="pushButton_C">
                                <property name="text">
                                    <string>C</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QPushButton" name="pushButton_1">
                                <property name="text">
                                    <string>1</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="3">
                            <widget class="QPushButton" name="pushButton_multiplication">
                                <property name="text">
                                    <string>X</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="1">
                            <widget class="QPushButton" name="pushButton_0">
                                <property name="text">
                                    <string>0</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QPushButton" name="pushButton_5">
                                <property name="text">
                                    <string>5</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QPushButton" name="pushButton">
                                <property name="text">
                                    <string>%</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="2">
                            <widget class="QPushButton" name="pushButton_9">
                                <property name="text">
                                    <string>9</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QPushButton" name="pushButton_hooks">
                                <property name="text">
                                    <string>()</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
